apps/web/src/services/alert-manager/components/AlertDetailSummaryAssignModal.vue

```diff
@@ -111,55 +122,47 @@ export default {
             }
         };
 
-        const onClickReassign = async () => {
+        const handleClickReassign = async () => {
             await reassignMember();
         };
 
-        const assignApiQueryHelper = new ApiQueryHelper()
-            .setPageStart(1).setPageLimit(15)
-            .setSort('resource_id', true);
-
-        let assignApiQuery = assignApiQueryHelper.data;
-
-        const listMemberInProject = async () => {
+        const getProjectUserData = async () => {
             try {
                 state.loading = true;
-                const { results, total_count } = await SpaceConnector.client.identity.project.member.list({
+                const params: ProjectGetRequestParams = {
+                    workspace_id: '', // TODO: workspace_id
```

Comment:
suggestion: I think it's not needed because workspace_id will be added by SpaceConnector level.